src/makeCalculator.js

```diff
@@ -38,6 +38,42 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+    // operate : (calback, arg) => {
+    //   calback(arg);
+    // },
```

Comment:
looks better to implement like this, create the methods inside the calculator object directly